.changeset/wide-eels-marry.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/platform-workspace': patch
+---
+
+Fixed Platform sandboxes retrying commands while the provider is still starting and recovering when the provider reports a destroyed sandbox.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)